tracee/tracee.go

```diff
@@ -423,11 +431,6 @@ func copyFileByPath(src, dst string) error {
 	return nil
 }
 
-func (t *Tracee) handleError(err error) {
```

Comment:
error handling doesn't feel like the kind of work that will benefit from parallelism. it might be better to process errors in the same execution flow as the code that caused it. wdyt?

Reply:
Well, not all errors have to go to the errors channel. We can handle errors locally if required.
In the case of error handling that use handleError - we just update an error counter in Tracee "class", and print the error - I thought it would be better to just let the sending go routing keep doing it's job, while the "processErrors" go routine will take care of the print and error counting (which may be time consuming).
I don't mind leaving it the way it was if you are not convinced